src/kernel/src/drivers/chipset/include/chipset.h

```diff
@@ -3,6 +3,7 @@
 
 #include <stdbool.h>
 #include <stdint.h>
+#include "uart.h"
 
 /*
  * Abstract chipset driver interface
```

Comment:
I just checked your branch and noticed the declared here function `uart_putc(char c, int channel)` is not defined or used anywhere. Should it be removed?

Reply:
You might be mistaken, uart_putc is removed here. Not added.

Reply:
Are you sure it is removed? I am talking about the one on line 13